inc/class-saml.php

```diff
@@ -6,6 +6,8 @@
 use function Pressbooks\Utility\str_remove_prefix;
 use function Pressbooks\Utility\str_starts_with;
 use PressbooksMix\Assets;
+use Pressbooks\Log\CloudWatchProvider as CloudWatchProvider;
+use Pressbooks\Log\Log;
```

Comment:
Here's the reason because this PR needs the [Pressbooks Plugin PR](https://github.com/pressbooks/pressbooks/pull/2104)